src/Faker/Provider/ms_MY/Person.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Faker\Provider\ms_MY;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} bin {{lastName}}',
+        '{{firstNameMale}} bin {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} bin {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{secondPersonalNameMale}} {{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{secondPersonalNameMale}} {{firstNameMale}} bin {{lastName}}',
+        '{{prefix}} {{firstNameMale}} bin {{lastName}}',
+        '{{prefix}} {{titleMale}} {{firstNameMale}} bin {{lastName}}',
+        '{{prefix}} {{titleMale}} {{secondPersonalNameMale}} {{firstNameMale}} bin {{lastName}}',
+        '{{prefix}} {{secondPersonalNameMale}} {{firstNameMale}} bin {{lastName}}',
+
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} binte {{lastName}}',
+        '{{firstNameFemale}} binti {{lastName}}',
+        '{{secondPersonalNameFemale}} {{firstNameFemale}} binte {{lastName}}',
+        '{{secondPersonalNameFemale}} {{firstNameFemale}} binti {{lastName}}',
+        '{{secondPersonalNameFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} binte {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} binti {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{secondPersonalNameFemale}} {{firstNameFemale}} binte {{lastName}}',
+        '{{titleFemale}} {{secondPersonalNameFemale}} {{firstNameFemale}} binti {{lastName}}',
+        '{{titleFemale}} {{secondPersonalNameFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{prefix}} {{titleFemale}} {{secondPersonalNameFemale}} {{firstNameFemale}} binte {{lastName}}',
+        '{{prefix}} {{titleFemale}} {{secondPersonalNameFemale}} {{firstNameFemale}} binti {{lastName}}',
+        '{{prefix}} {{secondPersonalNameFemale}} {{firstNameFemale}} binte {{lastName}}',
+        '{{prefix}} {{secondPersonalNameFemale}} {{firstNameFemale}} binti {{lastName}}',
+        '{{prefix}} {{firstNameFemale}} binte {{lastName}}',
+        '{{prefix}} {{firstNameFemale}} binti {{lastName}}',
+    );
+
+    /**
+     * @link https://en.wikipedia.org/wiki/List_of_Malaysians
+     */
+    protected static $firstNameMale = array(
+        'Sudirman', 'Awal', 'Asmawi', 'Azhar', 'Sean', 'Aziddin', 'Shanon', 'Hani', 'Mohsin', 'Aziz', 'Azmyl', 'Aznil',
```

Comment:
In this dataset, as well as in the other ones of this class, there is not enough data to make a good randomization. Can't you find more?